must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -67,3 +67,18 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+#Add ocs ready nodes to the list
+nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
+
+#collecting journal logs
+for node in ${nodes}; do
+    printf "collecting crash and journal logs  from node %s \n"  "${node}"
+    oc debug nodes/"${node}" << CMDS
+       chroot /host
+       cd var
+       journalctl -o verbose >> journal.log &
```

Comment:
I think you may need to wait for the completion of this cmd otherwise it will exit soon and we may not have complete info gathered.

Reply:
yes. Either don't send it into the background or insert a `wait` command in the next line.

Reply:
fixed